sequencer/src/lib.rs

```diff
@@ -520,18 +522,18 @@ mod test {
     async fn hotshot_test() {
         setup_logging();
         setup_backtrace();
-
-        TestMetadata {
-            overall_safety_properties: OverallSafetyPropertiesDescription {
-                num_successful_views: 10,
-                ..Default::default()
-            },
-            ..Default::default()
-        }
-        .gen_launcher::<SeqTypes, Node<network::Memory>>(0)
```

Comment:
It's unclear what needs to be implemented here in order to satisfy the compiler.

Reply:
The first argument of `gen_launcher` is `TYPES: NodeType`, so it seems like `SeqTypes` ought to be correct What error are you getting?

Reply:
```
error[E0599]: the method `run_test` exists for struct `TestRunner<SeqTypes, Node<Memory>>`, but its trait bounds were not satisfied
   --> sequencer/src/lib.rs:539:10
    |
171 |   pub struct SeqTypes;
    |   ------------------- doesn't satisfy `<_ as NodeType>::InstanceState = TestInstanceState`
...
530 | /         TestMetadata {
531 | |             overall_safety_properties: OverallSafetyPropertiesDescrip...
532 | |                 num_successful_views: 10,
533 | |                 ..Default::default()
...   |
538 | |         .launch()
539 | |         .run_test()
    | |         -^^^^^^^^ method cannot be called on `TestRunner<SeqTypes, Node<Memory>>` due to unsatisfied trait bounds
    | |_________|
    | 
    |
    = note: the following trait bounds were not satisfied:
            `<SeqTypes as hotshot_types::traits::node_implementation::NodeType>::InstanceState = hotshot_testing::state_types::TestInstanceState`

```

Reply:
Ohhh it seems we can only run the HotShot tests if we use their test types. TBH I'm not even sure this test is worth maintaining then. We have plenty of our own tests that run HotShot. You can delete this test.